decidim-participatory_processes/app/views/decidim/participatory_processes/admin/participatory_processes/_form.html.erb

```diff
@@ -128,6 +126,24 @@
     </div>
   </div>
 
+  <div class="card" data-component="accordion" id="accordion-taxonomies">
+    <div class="card-divider">
+      <button class="card-divider-button" data-open="true" data-controls="panel-taxonomies" type="button">
+        <%= icon "arrow-right-s-line" %>
+        <h2 class="card-title" id="taxonomies">
+          <%= t("taxonomies", scope: "decidim.participatory_processes.admin.participatory_processes.form") %>
+        </h2>
+      </button>
+    </div>
+    <div id="panel-taxonomies" class="card-section">
+      <% @form.taxonomy_filters.each do |filter| %>
+        <div class="row column">
+          <%= filter_taxonomy_items_select_field form, :taxonomies, filter %>
```

Comment:
this will become clear in the next PR if you'd allow me. The list is refactored to be in a tree form (if you really fill like you need it here I can port it). As this PR is just a step in the whole EPIC, and does not provide any real functionality reflected on the public side, I think is not worth it.